platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,41 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
+ Creates a set of options with the minimum required information.
+ 
  @param styleURL the style url to use
```

Comment:
Parameter documentation should start with a capital letter and end with a period. URL should be capitalized. Note that this differs from the prevailing code style in the Android SDK.

This is based on the documentation for `-[MGLMapView initWithFrame:styleURL:]`:

> URL of the map style to snapshot. The URL may be a full HTTP or HTTPS URL, a Mapbox URL indicating the style’s map ID (`mapbox://styles/{user}/{style}`), or a path to a local file relative to the application’s resource path. Specify `nil` for the default style.

(Can local styles be snapshotted?)

/ref https://github.com/mapbox/mapbox-gl-native/pull/9891#discussion_r137133807